<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simple SQLite Examples</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnExit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>520</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tab">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>431</width>
      <height>511</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabHome">
     <attribute name="title">
      <string>Home</string>
     </attribute>
     <widget class="QLabel" name="lblPhoto">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>0</y>
        <width>280</width>
        <height>280</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string>Photo</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lblStateName">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>280</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string>State Name</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lblCapitol">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>310</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string>Capitol</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lblEstablished">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>340</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string>Est.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lblCurrentIndex">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>400</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btnIndexBack">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>400</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="toolTipDuration">
       <number>2</number>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnIndexForward">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>400</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="toolTipDuration">
       <number>2</number>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabSearch">
     <attribute name="title">
      <string>Search</string>
     </attribute>
     <widget class="QLabel" name="lblSearchPageTitle">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>421</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Search Page</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtSearchStateName">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Enter State Name</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="pteSearchResults">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>371</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>390</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Index to Retrieve</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtSearchIndex">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>390</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btnGo">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>390</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Go</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabDataEntry">
     <attribute name="title">
      <string>Data Entry</string>
     </attribute>
     <widget class="QLabel" name="lblDataEntryPageTitle">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>421</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Data Entry Page</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lblPhotoSelected">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>280</width>
        <height>280</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSelectPhoto">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>310</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Select Photo</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtStateName">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>340</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Input State Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtCapitol">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>370</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Input Capitol City</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtEstablished">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>400</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Input Year Established</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSaveRecord">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>440</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Save Record</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>449</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>tab</tabstop>
  <tabstop>btnExit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
